src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -84,6 +85,23 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
               }
             />
           </RenderIf>
+          <RenderIf condition={cluster.status == 'installed'}>
+            <Alert
+              variant="info"
+              isInline
+              title={
+                <p>
+                  Add new hosts by generating a new Discovery ISO under your cluster's "Add hosts”
+                  tab on{' '}
+                  <a href={REDHAT_CONSOLE_OPENSHIFT} target="_blank" rel="noopener noreferrer">
+                    console.redhat.com/openshift <i className="fas fa-external-link-alt" />
```

Comment:
```suggestion
                    console.redhat.com/openshift <i className="fas fa-external-link-alt" />
```

Please reuse `<ExternalLinkAltIcon />` instead.

[example](
https://github.com/openshift-assisted/assisted-ui-lib/blob/0af7c458b5910df3764fe440066ff5f16659e154/src/common/components/ui/formik/PullSecretField.tsx#L38)